src/audio/codec_adapter/Kconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+menu "Codec Adapter codecs"
+	visible if COMP_CODEC_ADAPTER
+
+	config CADENCE_CODEC
+		bool "Cadence codec"
+		default n
+		help
+			Select for codecs which conforms to the Cadence API.
+			This will cause codec adapter component to include header
+			files specific to CADENCE base codecs.
+
+	config DUMMY_CODEC
+		bool "Dummy codec"
+		default n
+		help
+			Select for a dummy API codec implementation.
+			This will cause codec adapter component to include header
```

Comment:
The text under help must be indented with 2 spaces. 
So we need something like this:

``
<tab>help
<tab>ss
``

Reply:
Thanks. Done.